### PR TITLE
New package: twmn-git

### PR DESCRIPTION
**Pull request by sirikid**
First attempt, tips are welcome.

### CODE REVIEW

**Comment by cr6git** [comment]
Your template is completely wrong.

**Comment by sirikid** [comment]
Ok, I messed up with dependencies and archs. Anything else?

File: srcpkgs/twmn-git/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- mode: sh; -*-
+# Template file for 'twmn-git'
+pkgname=twmn-git
```
[the remark this answers is not in the document]
Comment (sirikid) [reply srcpkgs/twmn-git/template]: It's a bit tricky, twmn's author doesn't publish releases at all.

File: srcpkgs/twmn-git/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+version=90627.a892ca30c5
+revision=1
+_commit=5b92ac5d8c805a536211cb8dcee987247c0e6707
+wrksrc="twmn-$_commit"
+build_style=qmake
+only_for_archs="i686 x86_64 x86_64-musl"
```
[the remark this answers is not in the document]
Comment (sirikid) [reply srcpkgs/twmn-git/template]: But they are ¯\\\_(ツ)\_/¯

**Comment by newbluemoon** [comment]
To build the cross targets you have to add

```
if [ "$CROSS_BUILD" ]; then
	hostmakedepends+=" qt5-devel"
fi
```

Qt5 is a bit different in this respect. ;)

And please remove empty variables (`depends=""`).

**Comment by maxice8** [comment]
Please request upstream to make a proper tagged release

**Comment by sirikid** [comment]
Thank you all for participating! During the time I worked on this template, I realized that the program itself was written quite carelessly and, perhaps, it should not be included in the repository.

**Pull request closed without merging**